keepassxc-browser/keepassxc-browser.js

```diff
@@ -838,26 +849,42 @@ cipFields.getOverflowHidden = function(field) {
     return false;
 };
 
+cipFields.isVisible = function(field) {

```

Comment:
`aria-hidden` is a HTML5 specific property. `isVisible()` checks if the element is actually visible on the page and has dimensions. That's why these shouldn't be combined. But `getAriaHidden()` shouldn't be called if the element is not visible so I'll switch the call to the if-sentence instead of calling it every time.

Reply:
I'm aware about the technical differences, but from an API standpoint, it's both just "I want to see whether this element is really visible and I should fill it".

Reply:
I understand what you mean. The fix I just committed doesn't even run `getAriaHidden()` if `isVisible()` returns false. This prevents any extra checking of attributes or looping parent elements when not needed.

Reply:
Sure, but shouldn't it be part of isVisible() then and be checked if none of the previous tests already failed?